src/app/help/help.component.html

```diff
@@ -0,0 +1,142 @@
+
+          </li>
+
+          <li>
+              <h3>How to search images on Susper?</h3>
+              <p>You can search Susper for images, just like you can search for websites. For example, you can check out pictures of potential vacation destinations, or find an image to use in an upcoming presentation.</p>
+              <ol>
+                <li>Go to <a href="https://susper.com">susper.com</a></li>
+                <li>Search your query</li>
+                <li>Then select the Image tab, located below the search bar</li>
+              </ol>
+
+          </li>
+
+          <li>
+              <h3>How to block explicit content on Susper?</h3>
+              <p>You can filter explicit search results on Susper, like pornography, with the SafeSearch setting. SafeSearch isn’t 100% accurate. But it can help you avoid explicit and inappropriate search results on your phone, tablet, or computer.</p>
+              <ol>
+                <li>Go to <a routerLink="/preferences">Search Settings</a></li>
+                <li>Under "SafeSearch filters," check or uncheck the box next to "Turn on SafeSearch."</li>
+                <li>At the bottom of the page, select Save.</li>
+              </ol>
+
+          </li>
+
+          <li>
+              <h3>How to use voice search?</h3>
+              <p>You can use your voice to do searches by simply tapping on the microphone on the search bar and speaking your query.</p>
+
+          </li>
+
+          <li>
+              <h3>How to report a problem with Susper?</h3>
+              <p>If you're having a problem with Susper Search, or want to tell us your thoughts about how it's working, you can send a feedback report to Susper.</p>
+              <ol>
+                <li>Open our <a routerLink="/contact">Contact page.</a></li>
+                <li>Scroll down to the bottom of the page</li>
+                <li>Click on Contact Form</li>
+                <li>Fill out the information</li>
+                <li>Click Send</li>
+              </ol>
+
+          </li>
+
+          <li>
+              <h3>How to use Advanced Search?</h3>
+              <p>Narrow down search results for complex searches by using the Advanced Search page.</p>
+              <ol>
+                <li>Scroll down to the bottom of the page</li>
+                <li>Click on Settings, and then click on Advanced Search</li>
+                <li>Fill out the parameters</li>
+                <li>Click on Advanced Search button</li>
+              </ol>
+
+          </li>
+
+
+          <li>
+              <h3>How Susper works?</h3>
+              <p>Susper is a decentral Search Engine that uses the peer to peer system 'YaCy' and 'Apache Solr' to crawl and index search results.</p>
+              <p>For more information on how Susper works, visit our <a routerLink="/about">About Us</a></p>
+
+          </li>
+
+        </ul>
+
+
+
+
```

Comment:
Try to write clean code, remove extra unwanted spaces. Please follow it everywhere!